open-api.yaml

```
@@ -323,9 +323,9 @@ components:
             indexedDocuments:
               type: integer
               description: Number of documents finally indexed for documentAdditionOrUpdate task or a documentAdditionOrUpdate batch of tasks.
-            receivedDocumentsIds:
+            matchedDocuments:
```

```suggestion
            providedIds:
```

As discussed on Slack, we decided that it would be more correct to use this naming as it doesn't represent the number of documents that match but raw documents ids, the number of existing documents is not known at this point.